block-explorer/v3.yml

```diff
@@ -0,0 +1,643 @@
+      tags:
+        - Global snapshot
+      summary: Get rewards for given global snapshot
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of global snapshot to return rewards from
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Rewards for given global snapshot
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RewardTransaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{ordinal}/rewards:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get rewards for given global snapshot
+      parameters:
+        - name: ordinal
+          in: path
+          description: Ordinal of global snapshot to return rewards from
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        200:
+          description: Rewards for given global snapshot
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RewardTransaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /block/{hash}:
+    get:
+      tags:
+        - Block
+      summary: Get block by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of block to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Block for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Block'
+
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /transaction/{hash}:
+    get:
+      tags:
+        - Transaction
+      summary: Get transaction by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of transaction to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Transaction for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Transaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/latest/transaction:
```

Comment:
We should allow `search_before`, `search_after`, `limit` parameters there as well